x-pack/plugins/stack_connectors/server/connector_types/openai/openai.test.ts

```diff
@@ -276,11 +276,16 @@ describe('OpenAIConnector', () => {
     describe('invokeStream', () => {
       let stream;
       beforeEach(() => {
+        const chunk1 =
+          'data: {"object":"chat.completion.chunk","choices":[{"delta":{"content":"My"}}]}\ndata: {"object":"chat.completion.chunk","choices":[{"delta":{"content":" new"}}]}';
+
         stream = createStreamMock();
-        stream.write(`data: ${JSON.stringify(mockResponse.data)}`);
+        stream.write(chunk1);
+        // .write(`data: ${JSON.stringify(mockResponse.data)}`);
         mockRequest = jest.fn().mockResolvedValue({ ...mockResponse, data: stream.transform });
         // @ts-ignore
         connector.request = mockRequest;
+        stream.complete();
```

Comment:
@dgieselaar if i don't have this, the `flush` does not get called and `transforms the response into a string` test fails